### PR TITLE
Feature - Update Classifier Mappings for New conditions

### PR DESCRIPTION
**Pull request by lukey-luke**
## What was the problem?

This PR is part 2 of 2 parts in order to update the lookup table mappings.

This PR includes updates to the ConditionDropdownLookupTable which is used for mapping autosuggestions and free text to classifications.
This PR also includes changes to use a new .csv file for capturing dropdown coverage in the datadog logs.
 - this was moved to a separate csv file in order to improve quality of life for folks doing the research for these mappings

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/459

## How does this fix it?[^1]

csv's for lookup tables are now split into three directories
 - `domain-cc/cc-app/src/python_src/util/data/condition_dropdown_coverage`
   - this is what is used to determine what is logged as `is_in_dropdown` in the datadog logs
 - `domain-cc/cc-app/src/python_src/util/data/condition_dropdown_lookup_table`
   - this is what maps autosuggestion and free text to classifications
 - `domain-cc/cc-app/src/python_src/util/data/dc_lookup_table`
   - this remains the same - diagnostic codes are mapped to classifications

## How to test this PR
 - `cd domain-cc/cc-app`
 - `pytest`
 - Note lookup table size has increased. There are 50 new mappings

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by tyler-spangler6 — APPROVED** [review]
LGTM

File: domain-cc/cc-app/src/python_src/pydantic_models.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
+    diagnostic_code: Optional[int] = (
+        None  # only required for claim_type: "claim_for_increase"
+    )
     claim_type: str = "claim_for_increase"
-    contention_text: Optional[
-        str
-    ] = None  # marked optional to retain compatibility with v1
+    contention_text: Optional[str] = (
+        None  # marked optional to retain compatibility with v1
+    )
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/pydantic_models.py:15]: these are formatting changes only triggered by commit hook

**Review by msnwatson — APPROVED** [review]
No review necessary from VRO but LGTM

**Comment by nelsestu** [comment]
Preparing questions...

**Comment by nelsestu** [comment]
Mainly, one question I had was regarding the versioning of the CSV files. For example was the update to `CONDITION_DROPDOWN_COVERAGE_VERSION` from a non-existent version to "v0.1" creating file versioning strategy?

Understanding the versioning a bit more might help with my comprehension about the PR. But otherwise this LGTM.

**Review by nelsestu — APPROVED** [review]
Looks great.  Answering the one question i added is completely optional, and should not delay this PR. Thanks!

**Comment by tyler-spangler6** [comment]
@nelsestu, yes we are starting a new versioning for this csv file. We previously used the condition_dropdown_lookup_table csv files to build the dropdown list to use for logging. We are breaking this out into two separate csv files to hopefully make it easier to update these without affecting other tables/lists.

**Pull request merged**